proto/ibc/core/channel/v2/channel.proto

```diff
@@ -42,3 +42,34 @@ message Payload {
   // the raw bytes for the payload.
   bytes value = 3;
 }
+
+// Acknowledgement contains a list of all ack results associated with a single packet.
+message Acknowledgement {
```

Comment:
Details around this might change after https://github.com/cosmos/ibc/pull/1149#pullrequestreview-2343064642 is done